lib/cfn-model/model/ec2_network_acl_entry.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require_relative 'model_element'
+
+class AWS::EC2::NetworkAclEntry < ModelElement
+  def initialize(cfn_model)
```

Comment:
if you are defining an explicit model element then you should likely have explicit attribute(s).  you shouldn't "shadow" attributes that are created and populated automagically by the model..... but add attributes with different names for anything that is synthesized